### PR TITLE
My First Try 

### PR DESCRIPTION
**Pull request by nullllllion**
Tried to build a google home page for improving my html and css skills...

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
-    <h1>Hello there! 👋🏼</h1>
-  </body>
-</html>
\ No newline at end of file
+        <div class="navbar">
+            <ul>
+                <li>Gmail</li>
+                <li>Images</li>
+                <li><button id="signIn_button">Sign In</button></li>
+            </ul>
```
Comment (MichelLaufer) [inline code/index.html:24]: Nice with the navbar!

File: code/index.html [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
+        </div>
+        <div class="main_page">
+            <h2>
+                Google
+            </h2>
+            <input type="text" id="input">
+            <div class="button">
+                <button class="google_btn">Google Search</button>
+                <button class="google_btn">I'm Feeling Lucky</button>
+            </div>
```
Comment (MichelLaufer) [inline code/index.html:34]: It looks good with the searchbar and buttons! 👍 

A couple of suggestions are to change the class name from `"button"` to for example `"button-container"` and from `"google_btn"` to `google-btn` (using kebab-case for CSS).

File: code/style.css [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+    color: #fff;
+    border: 10px solid  rgb(72, 116, 236);
+    border-radius: 3px;
+    text-decoration: none;
+}
+#signIn_button:hover{
+    background-color:  rgba(77, 127, 253, 0.8);
+    border: 10px solid  rgba(77, 127, 253, 0.849);
+    text-decoration: none;
+}
```
Comment (MichelLaufer) [inline code/style.css:35]: Nice that you use hover and change the background color! 
You may also add `cursor: pointer;`

File: code/style.css [hunk 4]
```diff
@@ -1,5 +1,93 @@
...
+.main_page{
+    position: absolute;
+    display: flex;
```
Comment (MichelLaufer) [inline code/style.css:38]: Nice with flexbox!

File: code/style.css [hunk 5]
```diff
@@ -1,5 +1,93 @@
...
+}
+.google_btn{
+    cursor: pointer;
+    padding: 10px;
+    margin: 5px;
+    font-size: 14px;
+    color: rgba(34, 34, 34, 0.774);
+background-color: #f8f8f8;
+background-color: rgb(248, 248, 248);
+border: none;
```
Comment (MichelLaufer) [inline code/style.css:64]: Here I would move the text to the right for consistent indentation.

**Review by MichelLaufer — COMMENTED** [review]
Very good job! 👍 I think that the searchbar with the title and buttons looks nice
and I also like the navbar. The indentation in the code is very good! 

I have made some minor suggestions below.

Keep up the good work!

File: code/style.css (hunk 5 above)
Comment (nullllllion) [reply code/style.css:64]: thank you so much for your appreciation.

**Pull request closed without merging**